localization/english/culture/wc_basic_innovations_l_english.yml

```diff
@@ -0,0 +1,27 @@
+﻿l_english:
+
+ ##############
+ # Tribal Era #
+ ##############
+ 
+ ######################
+ # Early Medieval Era #
+ ###################### 
+ 
+ innovation_animal_war_training:0 "Animal War Training"
+ innovation_animal_war_training_desc:0 "Through skills taught to us by the best beastmasters, we can discipline, train and communicate with the animals around us."
+ 
+ innovation_rogues_guild:0 "Rogues Guild"
+ innovation_rogues_guild_desc:0 "Guilds made up of rogues, thieves, and assassins are a presence in many major cities, though rarely officially acknowledged. Despite that, however, their unique talents, blending stealth, skill, and ruthlessness, make them a formidable asset for any nation."
+ 
+ #####################
+ # High Medieval Era #
+ #####################
+ innovation_rogues_school:0 "School of Roguery"
+ innovation_rogues_school_desc:0 "To be trained as a rogue is a difficult, frequently deadly quest. Students are taught the use of tools, combat techniques, evasion, and perhaps most infamously, assassination. Schools specializing in the art of roguery are rare, but almost uniformly produce skilled agents, should one overlook the numerous failed students."
+ 
+ #####################
+ # Late Medieval Era #
+ #####################
+ innovation_secret_police:0 "Secret Police"
```

Comment:
I don't think police exists on Warcraft. SI:7 for example, is described as secretive **military force** of Stormwind's sovereign monarch. I think something like "Spy Agency" fits better.